profilegen/src/res/templates/spec_template.template

```diff
@@ -0,0 +1,30 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+require_relative 'lib/client'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'azure_mgmt_profile_<%= profile_version %>'
+  spec.version       = '<%= profile_ruby_version %>'
+  spec.authors       = 'Microsoft Corporation'
+  spec.email         = 'azrubyteam@microsoft.com'
+  spec.description   = 'Microsoft Azure Management Profile'
+  spec.summary       = 'Official Ruby client library to consume Microsoft Azure Management profile.'
+  spec.homepage      = 'https://aka.ms/azure-sdk-for-ruby'
+  spec.license       = 'MIT'
+
+  spec.files         = Dir['LICENSE.txt', 'lib/**/*']
+  spec.require_paths = ['lib']
+
+  spec.required_ruby_version = '>= 2.0.0'
+
+  spec.add_development_dependency 'bundler', '~> 1.9'
+  spec.add_development_dependency 'rake', '~> 10'
+  spec.add_development_dependency 'rspec', '~> 3'
+  spec.add_development_dependency 'dotenv', '~> 2'
```

Comment:
did you find out whether we need `dotenv` or not ?

Reply:
"Shim to load environment variables from .env into ENV in development.". We would need this for specs.